### PR TITLE
Allow to select the benchmark to run

### PR DESCRIPTION
**Pull request by dunglas**
Example:

`$ bin/benchmark -i100 -sSymfonyGetSetNormalizer`

Convenient to debug.

### CODE REVIEW

File: src/Command/BenchmarkCommand.php [hunk 1]
```diff
@@ -46,24 +47,39 @@ protected function configure()
             ->setName('benchmark')
             ->addOption('iteration', 'i', InputArgument::OPTIONAL, 'Number of iteration(s)', 1)
             ->addOption('horizontal-complexity', 'hc', InputArgument::OPTIONAL, 'Horizontal data complexity', 1)
-            ->addOption('vertical-complexity', 'vc', InputArgument::OPTIONAL, 'Vertical data complexity', 1);
+            ->addOption('vertical-complexity', 'vc', InputArgument::OPTIONAL, 'Vertical data complexity', 1)
+            ->addOption('serializer', 's', InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY, 'the serializer(s) to use', null);
```
Comment (bendavies) [inline src/Command/BenchmarkCommand.php]: InputArgument::OPTIONAL?

**Comment by goetas** [comment]
:+1:

File: src/Command/BenchmarkCommand.php [hunk 2]
```diff
@@ -46,24 +47,39 @@ protected function configure()
             ->setName('benchmark')
             ->addOption('iteration', 'i', InputArgument::OPTIONAL, 'Number of iteration(s)', 1)
```
Comment (bendavies) [inline src/Command/BenchmarkCommand.php:48]: the `InputArgument` usage for the rest of the options is wrong, right?

Comment (dunglas) [reply src/Command/BenchmarkCommand.php:48]: Indeed, but as the value of the constants are the same, it works :)
A PR to fix this would been appreciated.

Comment (goetas) [reply src/Command/BenchmarkCommand.php:48]: yes, pre-existing bug! :smiley:

**Pull request merged**